### DISCUSSION TITLE
Problem getting data from API with Ajax request

### DISCUSSION THREAD

**Discussion by tomasnihlen** [cables-gl/cables_docs#379] [Answers & Questions] [answered]
Hi!

I am trying to get data from a public API into cables.gl but I get this error message:

Error: Problem while loading json:
SyntaxError: Unexpected end of JSON input

The API URL looks like this:
http://api.kolada.se/v2/data/kpi/N00945/municipality/1860/year/2009

The data is null and the response is null. But I can open the link in my browser and it should be in regular json format.

Could anybody help me figure this out? I am a beginner when it comes to using API:s.

All help is much appreciated! /Tomas

**Comment by steam0r** [comment, marked as answer]
try https instead of http

also keep cors in mind: https://cables.gl/docs/4_export_embed/cors/cors

**Reply by tomasnihlen** [reply]
Thanks! Changing to https actually solved it!

**Comment by pandrr** [comment]
learn to use dev-tools: open them and see the errors
